bin/main.ml

```diff
@@ -17,7 +17,7 @@ let all : _ Cmdliner.Cmd.t list =
     ; Print_rules.command
     ; Utop.command
     ; Promotion.promote
-    ; Printenv.command
+    ; command_alias Printenv.command Printenv.term "printenv"
```

Comment:
I'd pick a different name. {describe,show} + print sounds very redundant. `env` alone sounds better.

Reply:
This is the old command which was called `dune printenv`, the new one is `dune show env`.